src/main/java/seedu/address/model/event/EventTitle.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.event;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Event's title in the address book.
```

Comment:
```suggestion
 * Represents an Event's title in the address book.
```
Minor grammar change!